### PR TITLE
Create culture-war-open-source.md

### PR DESCRIPTION
**Pull request by ntnsndr**

### CODE REVIEW

File: content/blog/culture-war-open-source.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
++++
+
+The challenges to open source\'s \"noble neutrality.\"
+
+[Nathan Schneider](https://nathanschneider.info/)   
+April 15, 2020
+
+Perhaps you remember Eric S. Raymond from his memorable essay "[The
+Cathedral and the
+Bazaar](http://www.catb.org/esr/writings/cathedral-bazaar/)," or his[
```
Comment (CoralineAda) [inline content/blog/culture-war-open-source.md]: It looks like you have something weird going on with your links. I'm seeing this:

```[
](http://catb.org/~esr/faqs/smart-questions.html#not_losing)[advice](http://catb.org/~esr/faqs/smart-questions.html#not_losing)```
```
(notice how there is an empty anchor, a URL, then an anchor, and the same URL?

Should be like this:

```[advice](http://catb.org/~esr/faqs/smart-questions.html#not_losing)```

And it's showing as raw links. I think for links it should be just `[anchor text](http://…)`

File: content/blog/culture-war-open-source.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
++++
+title = ""The Ethical Source Working Group Blog"
```
Comment (CoralineAda) [inline content/blog/culture-war-open-source.md]: Eep, typo! The extra `"` at the beginning is causing the preview build to fail.

**Review by CoralineAda — COMMENTED** [review]
Go ahead and close the other PR and add the link from the main blog page to this one.

Thanks for this!

File: content/blog/culture-war-open-source.md (hunk 1 above)
Comment (ntnsndr) [reply content/blog/culture-war-open-source.md]: Eeek sorry! I didn't notice that got screwed up in the pandoc conversion. Will fix.

File: content/blog/culture-war-open-source.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
++++
+title = "The Ethical Source Working Group Blog"
+heading = "The Culture War in Open Source is On"
++++
+
+The challenges to open source\'s \"noble neutrality.\"
```
Comment (CoralineAda) [inline content/blog/culture-war-open-source.md]: No need to escape the single and double quotes

Comment (ntnsndr) [reply content/blog/culture-war-open-source.md]: Oh, sorry—just removed that line, which doesn't fit the format here. Those were the only escape backslashes I could find. Again, sorry—an excess introduced by pandoc in conversion.

Just committed to the fork.

**Review by CoralineAda — APPROVED** [review]
Awesome, thank you!

**Pull request merged**